### DISCUSSION TITLE
What version to use

### DISCUSSION THREAD

**Discussion by balajeek** [blakeblackshear/frigate#1399] [General]
I have HP Elite desk mini computer with Intel® Core™ i5-6500T CPU
when i tried this in my docker-compose:  image: blakeblackshear/frigate, it says mainfest not found, not sure what version of arch i should be using, if someone can guide or send me the link, me would be great.
thanks.

**Comment by joselito11** [comment]
blakeblackshear/frigate:stable-amd64
https://blakeblackshear.github.io/frigate/installation#docker
